topics/web/serializers/example1/main.go

```diff
@@ -26,6 +26,7 @@ func useStreams() {
 		LastName:  "Walker",
 	}
 
+	fmt.Print("json output: ")
```

Comment:
I'd rather not have these labels. Remove this line please

Reply:
Agree